packages/zend-db/library/Zend/Db/Adapter/Mysqli.php

```diff
@@ -343,6 +343,14 @@ protected function _connect()
         if (!empty($this->_config['charset'])) {
             mysqli_set_charset($this->_connection, $this->_config['charset']);
         }
+
+        if (!isset($this->_config['report_mode']) || PHP_VERSION_ID >= 80100) {
+            mysqli_report(
+                isset($this->_config['report_mode'])
+                    ? $this->_config['report_mode']
+                    : MYSQLI_REPORT_OFF
+            );
+        }
```

Comment:
I suppose this should not be added since exceptions are being handled now?

Reply:
right 👍  fixed